pcl.orogen

```diff
@@ -1,7 +1,16 @@
 name "pcl"
 version "0.1"
 
-using_library 'pcl_common-1.7'
+for pk in Utilrb::PkgConfig.each_package() do
```

Comment:
Python developer, are we ? ;-)

Assuming that the regex param indeed does not work, a much more idiomatic ruby would be

~~~
pkg = Utilrb::PkgConfig.each_package.find { |name| /^pcl_common/.match(name) }
using_library pkg
~~~

Just realized that you don't have to extract the version string, since `using_library` wants the pkg-config package name anyways